tests/unit/lib/telemetry/test_telemetry.py

```diff
@@ -121,15 +121,27 @@ def test_must_swallow_timeout_exception(self, requests_mock):
         #
 
         requests_mock.exceptions.Timeout = requests.exceptions.Timeout
+        requests_mock.exceptions.ConnectionError = requests.exceptions.ConnectionError
```

Comment:
Is this needed here if the this test is only for Timeout exception?

Reply:
Yeah lbecause of how Python exception mocking works